observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,73 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6_{context}"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3868[Network Observability Operator 1.6.0]
```

Comment:
Link doesn't work. Should we still be merging this?

Reply:
Yeah its only merging to a staged feature branch until GA which is Monday.

Reply:
Then errata links will work 🤞